### ISSUE TITLE
concurrency of the trustee?

### ISSUE THREAD

**Issue by ccxiaop** [confidential-containers/trustee#470]
Trustee should support concurrency. What is the approximate concurrency of the trustee?

**Comment by Xynnn007** [comment]
Yes we once made a PR for this, see https://github.com/confidential-containers/trustee/pull/258#issuecomment-1871733541

Currently for TDX the bottle neck is underlying `intel-tee-quote-verification-rs` crate. For SNP like platforms, it's performance is directly proportional to the given resource of CPUs.
